creditcoin-js/src/utils.ts

```diff
@@ -15,3 +16,32 @@ export const utils = (api: ApiPromise) => {
         signAccountId: (signer: Wallet, accountId: AccountId) => signAccountId(api, signer, accountId),
     };
 };
+
+const isDispatchError = (instance: any): instance is DispatchResult => {
```

Comment:
Why put this in `utils` instead of `common`? or maybe make an `error.ts` file?

Reply:
Good point. I normally use `utils` as 'it doesn't fit anywhere else'. I didn't create new files as I already felt like I was stretching the PR's scope enough. 'common' would have been more appropriate, but it skipped my mind.

Reply:
Lets move it to `common.ts` then, I was using `utils` for utils on CreditcoinApi.